tools/release/01_bump_crate_versions.ts

```diff
@@ -8,10 +8,6 @@ const repo = workspace.repo;
 const cliCrate = workspace.getCliCrate();
 const originalCliVersion = cliCrate.version;
 
-// update the std version used in the code
-console.log("Updating std version...");
-await updateStdVersion();
```

Comment:
Moved this below because on line 23 (now 19) it may prompt when running the script manually and the user might cancel the prompt.